### DISCUSSION TITLE
Missing link to React Virtual v2 documentation or migration guide

### DISCUSSION THREAD

**Discussion by diondiondion** [TanStack/virtual#427] [Q&A] [answered]
Hi, we're still using React Virtual 2.10.4, and I'm struggling to find a link to the documentation for that version of the library, or a migration guide that highlights the changes between version 2 and TanStack Virtual. Tried navigating back in the repo's commit history, but got lost there, too.

Are the old docs still online somewhere, am I missing something? 🙈

**Comment by piecyk** [comment, marked as answer]
Hi, yes, here is the link https://react-virtual-v2.tanstack.com/

**Reply by diondiondion** [reply]
Thanks, much appreciated. May I suggest adding this link to the sidebar of the v3 docs, or some other place where it can easily be found? I really tried my best to track this down, but wasn't able to, I'm sure others would have this issue, too.

**Reply by makker** [reply]
The link doesn't seem to work anymore. Just takes to v3 site, but no migration guide v2 -> v3 there.  Version 3 site does have a link to v2 docs (https://github.com/TanStack/virtual/tree/v2/docs/src/pages/), but nothing about migrating in there.

**Reply by jaredmcateer** [reply]
Took over a legacy project that uses react-virtual v2 and want to migrate it, this is very demoralizing finding that even the v2 docs repo link has been overwritten

**Comment by smellai** [comment]
I confirm the link redirects to new docs

**Comment by sirusdas** [comment]
Can anyone please help with migration from react-virtual@2.10.4 to latest as npm warns this version only supports up to react 17.
I have upgraded my react to v18. Please help!
